change/@internal-react-composites-ae9ef7c5-4a19-4aec-8f5d-f146b82fc48c.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "patch",
+  "comment": {
```

Comment:
no comment has been added in this change file, normally they are of the format:
![image](https://user-images.githubusercontent.com/2684369/150178945-2f5b25f7-5e04-400e-a89f-b15147dbf841.png)

(not sure how yours got title and value fields)

Can you update this? The reason for the change files is we use them to auto generate the changelog for releases

Reply:
Fixed! The comment did not stick when I did rush changelog. Will make sure to double check for next time. Thank you